pyirf/coordinates.py

```diff
@@ -0,0 +1,62 @@
+import astropy.units as u
+from astropy.coordinates import SkyCoord, SkyOffsetFrame, angular_separation, position_angle
+
+__all__ = [
+    'fov_coords_lon_lat',
+    'fov_coords_theta_phi'
+]
+
+
+def fov_coords_lon_lat(lon, lat, pointing_lon, pointing_lat):
```

Comment:
I'd put a `gadf_` in front here, other standards might use different definitions of these coordinates. Also add links to the corresponding gadf documentation.